src/glog/logging.h.in

```diff
@@ -430,6 +430,9 @@ DECLARE_int32(stderrthreshold);
 // Set whether the log prefix should be prepended to each line of output.
 DECLARE_bool(log_prefix);
 
+// Set whether the log prefix should be contain the year in each line of output.
```

Comment:
`should be contain` has incorrect grammar.
`each line of output` looks redundant, since `log_prefix` variable (above) controls whether the log prefix is used or not, and its description already mentions `each line of output`.
I would suggest this phrasing:
```
Set whether the year should be included in the log prefix.
```

Reply:
I modified the text according to your suggestion.